web_app/contract_tools/mixins/alert.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains the alert mixin class.
+"""
+from typing import List
+from web_app.db.crud import UserDBConnector
+from web_app.db.models import Status
+from web_app.contract_tools.mixins.dashboard import DashboardMixin
+from web_app.contract_tools.mixins.custom_exception import HealthRatioLevelLowException
+
+class AlertMixin:
+    """
+    Mixin class for alert related methods.
+    """
+
+    def check_users_health_ratio_level(self) -> None:
+        """
+        Check the health ratio level for all users with an OPENED position.
+        If a user's health ratio level is lower than 1.1, raise a `HealthRatioLevelLowException`.
+        """
+
+        users = UserDBConnector().get_all_users_with_opened_position()
+
+        if users:
```

Comment:
why do you need it here, if any users were found it will return empty list, please remove it